modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3947,6 +3947,11 @@ public CodegenProperty fromProperty(String name, Schema p, boolean required, boo
             List<Object> _enum = p.getEnum();
             property._enum = new ArrayList<>();
             for (Object i : _enum) {
+                // raw null values in enums are unions for nullable
+                // atttributes, not actual enum values, so we remove them here
+                if (i == null) {
+                    continue;
```

Comment:
thanks for the PR 👍 

shall we also set `property.isNullable` to `true` when null is found?

Reply:
yes, I think that would make sense actually.

Reply:
can we do it in a separate PR?

Reply:
sure

Reply:
```suggestion
                    property.isNullable = true;
                    continue;
```
or here you go :)